plat/rockchip/rk3368/drivers/pmu/pmu.c

```diff
@@ -0,0 +1,436 @@
+/*
+ * Copyright (C) 2016, Fuzhou Rockchip Electronics Co., Ltd.
```

Comment:
Thanks for changing the license text but could you please ensure the whole boilerplate text conforms to the rest of the codebase? So this should say "ARM Limited and Contributors" here. Note, that as a TF contributor, you are still recognised as the copyright holder.